### DISCUSSION TITLE
twinBASIC BETA 61

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#939] [Releases]
- added: *highly experimental* container support (frames etc).  Tab-key navigation/focus not yet working here.
- improved: added Container property to all controls
- fixed: support for ActiveX-based option buttons (e.g. VBCCR.OptionButtonW)

<hr /><em>This discussion was created from the release <a href='https://github.com/WaynePhillipsEA/twinbasic/releases/tag/beta-61'>twinBASIC BETA 61</a>.</em>

**Comment by camomille22003** [comment]
**It's magic for a first draft**

![Animation](https://user-images.githubusercontent.com/33807881/175567304-cc02122b-a6d5-4164-adef-60b3a0a9eb06.gif)

**My problem Start [Wait] Stop started again (Works well by "Preview")**

**Comment by camomille22003** [comment]
Here is an example made with version 61
Controls placed on tabs other than zero must not be positioned by program (only in property) otherwise they appear on all tabs.

![Animation6](https://user-images.githubusercontent.com/33807881/175779421-c5aad3a6-045e-4348-9d7e-8ed8816820a4.gif)

**Comment by dmrvb** [comment]
Is this just a known limitation of the SStab?  you can locate controls from code as long as the tab is visible, but Not if another tab is visible. The SStab has all controls visible but moves the ones from inactive tabs to a position out of range of the screen (left=-30000 or something like that) so if you set left its brings them back visible.  I wish some clever soul here (Krool????)  would write us a better version of SStab.

**Reply by camomille22003** [reply]
> Is this just a known limitation of the SStab? you can locate controls from code as long as the tab is visible, but Not if another tab is visible. The SStab has all controls visible but moves the ones from inactive tabs to a position out of range of the screen (left=-30000 or something like that) so if you set left its brings them back visible. I wish some clever soul here (Krool????) would write us a better version of SStab.

No, it's a limitation of version 61 that if you modify the position of the controls (by program) of the tabs other than zero, they anchor to the zero tab which serves as a reference, in VB6 this does not happen.

**Reply by EduardoVB** [reply]
> Is this just a known limitation of the SStab? you can locate controls from code as long as the tab is visible, but Not if another tab is visible. The SStab has all controls visible but moves the ones from inactive tabs to a position out of range of the screen (left=-30000 or something like that) so if you set left its brings them back visible. I wish some clever soul here (Krool????) would write us a better version of SStab.

I made a comment in #941

**Comment by camomille22003** [comment]
Example if we add the position of button 2,
it brings up buttons 1 and 4

![Image5](https://user-images.githubusercontent.com/33807881/175783154-b969ba43-25f5-4723-b746-c132c09b72a0.jpg)
